### ISSUE TITLE
matplotlib installation issue inside vittualenv

### ISSUE THREAD

**Issue by basaks** [GeoscienceAustralia/PyRate#149]
Document this as well  somwhere.

If matplotlib does not play well in a virtualenv, use this:

`sudo apt-get -y build-dep matplotlib`

Then install matplotlib again in your virtualenv.

I also make a note of this [here](https://gist.github.com/basaks/b33ea9106c7d1d72ac3a79fdcea430eb), which is linked in the readme.

**Comment by mcgarth** [comment]
@basaks is this in relation to run-time messages like the following:

```Matplotlib is building the font cache using fc-list. This may take a moment.
  warnings.warn('Matplotlib is building the font cache using fc-list. This may take a moment.')```

**Comment by basaks** [comment]
@mcgarth , not exactly.
`matplotlib` does not install properly in a `virtualenv`, even though if might be working from the system installed `matplotlib`.

**Comment by basaks** [comment]
@mcgarth to decide best way to do this. A readme link to this may be sufficient: https://gist.github.com/basaks/b33ea9106c7d1d72ac3a79fdcea430eb#gistcomment-2022012

**Comment by mcgarth** [comment]
resolved in commit 8fabc0d by adding info to README file and installation documentation
